be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -964,6 +974,122 @@ void VNodeChannel::mark_close() {
     _eos_is_produced = true;
 }
 
+FetchAutoIncIDExecutor::FetchAutoIncIDExecutor() {
+    ThreadPoolBuilder("AsyncFetchAutoIncIDExecutor")
+            .set_min_threads(config::auto_inc_fetch_thread_num)
+            .set_max_threads(config::auto_inc_fetch_thread_num)
+            .set_max_queue_size(std::numeric_limits<int>::max())
+            .build(&_pool);
+}
+
+AutoIncIDRequestHandlerExecutor::AutoIncIDRequestHandlerExecutor() {
+    ThreadPoolBuilder("AutoIncIDRequestHandlerExecutor")
```

Comment:
why here need tow thread pool?

Reply:
One for RPC and the other for handling requestss from `VOlapTableSink`. Putting these two kinds of threads in one thread pool may cause deadlock.